ptbcontrib/roles/README.md

```diff
@@ -51,20 +51,13 @@ async def post_init(application):
     
     # This will be accessible by anyone except my_role_1 and my_role_2
     application.add_handler(RolesHandler(SomeHandler(...), roles=~my_role_1))
-    
-    # This will be accessible only by the admins of the group the update was sent in
-    application.add_handler(RolesHandler(SomeHandler(...), roles=roles.chat_admins))
-    
-    # This will be accessible only by the creator of the group the update was sent in
-    application.add_handler(RolesHandler(SomeHandler(...), roles=roles.chat_creator))
 
     # You can compare the roles regarding hierarchy:
-    roles.ADMINS >= roles['my_role_1']  # True
-    roles.ADMINS >= roles['my_role_2']  # True
+    roles['my_role_1'] >= roles['my_role_2']  # True
     roles['my_role_1'] < roles['my_role_2']  # False
-    roles.ADMINS >= Role(...)  # False, since neither of those is a parent of the other
+    roles['my_role_1'] >= Role(...)  # False, since neither of those is a parent of the other
```

Comment:
It's still possible to compare the admin role.
it was just renamed `admins` instead of `ADMINS`
i.e 
```python
roles.admins >= Role(...)    # False
```